rules/forwarding_rules.yaml

```diff
@@ -12,35 +12,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-rules:
-  - name: Rule Name Example
-    target: Forwarding Rule Target Example
-    mode: whitelist
-    load_balancing_scheme: EXTERNAL
-    ip_protocol: ESP
-    ip_address: "198.51.100.46"
-
-  - name: TCP LB For Internal App 
-    target: https://www.googleapis.com/compute/v1/projects/THEPROJECT/regions/us-central1/THELB/FWD_RULE_NAME
-    mode: whitelist
-    load_balancing_scheme: INTERNAL
-    port: 8080
-    ip_protocol: TCP
-    ip_address: "198.51.100.23"
-
-  - name: UDP LB for External VPN
-    target: https://www.googleapis.com/compute/v1/projects/THEPROJECT/regions/us-central1/THELB/FWD_RULE_NAME
-    mode: whitelist
-    load_balancing_scheme: EXTERNAL
-    port_range: 4500-4500
-    ip_protocol: UDP
-    ip_address: "198.51.100.99"
-
-  - name: TCP LB for External VPN
-    target: https://www.googleapis.com/compute/v1/projects/THEPROJECT/regions/us-central1/THELB/FWD_RULE_NAME
-    mode: whitelist
-    load_balancing_scheme: EXTERNAL
-    port_range: 4500-4500
-    ip_protocol: TCP
-    ip_address: "198.51.100.35"
+#rules:
```

Comment:
By commenting all this out, will it break the forwarding rules scanner?

Reply:
Based on the code, it shouldn't break anything. I will deploy and test it after the data model bug is fixed because at this point we are not able to run scanner without data model.